rero/rero-ils
# docker: reduce image size of 500MB

* deletes Dockerfile.base
* merges Dockerfile.base and Dockerfile
* improves Dockerfile layer to reduce size of 500MB
* uses Debian buster as default image

Co-Authored-by: Olivier DOSSMANN <git@dossmann.net>

## Why are you opening this PR?

To reduce docker images size for deployment.

## How to test?

```
docker build -t rero/rero-ils:slim .
docker images|grep rero
```

Then compare size of different rero-ils images (rero/rero-ils:latest and rero/rero-ils:slim for an example).

Finally, run `docker-compose -f docker-compose.full.yml up -d` to check

## Code review check list
